### DISCUSSION TITLE
Updating Bloomberg Formulas/ ribbon functions

### DISCUSSION THREAD

**Discussion by HLaksmana** [dfinke/ImportExcel#1519] [Q&A]
Hi,

I am currently trying to refresh an RTD Bloomberg function (BDH) in Excel via Powershell. I've found solutions using vba but haven't found any resources that work for OpenOfficeXML. The standard calculate option for Excel formulas does not work for Bloomberg formulas. When this is done manually, the user has to go to the Bloomberg tab in the ribbon menu and click refresh or use a ribbon shortcut.

Has anyone used BBG with Powershell/ Import-Excel before?

**Comment by dfinke** [comment]
That won't work. The ImportExcel is not connected to the running instances of Excel. It only operates on the xlsx. VBA is the way. PowerShell using the `New-Object -COM` approach is possible.
